token-lending/program/src/processor.rs

```diff
@@ -2197,6 +2201,71 @@ fn process_update_reserve_config(
     Ok(())
 }
 
+#[inline(never)] // avoid stack frame limit
+fn process_redeem_fees(program_id: &Pubkey, accounts: &[AccountInfo]) -> ProgramResult {
```

Comment:
Should we restrict who can execute this function for security as its protocol specific? E.g only allowed by the solend keypair

Reply:
I think it's not needed because the function sends the fees to reserve.config.fee_receiver only, so it doesn't really matter who triggers this instruction.

Reply:
i think we specifically dont want to limit it because want to set up a crank to call it once a day from a non essential address